docs/06-CMS-Usage.md

```diff
@@ -3,7 +3,8 @@
 See the Search admin in the CMS
 
 ## Features
+
 - See if there are any applicable dirty objects to re-index
 - See the latest Solr logs
-- Add, remove or edit Synonyms
-    - Note that YML based synonyms can not be edited from the CMS
+- Add, remove or edit synonyms
+    - Note that YML-based synonyms can not be edited from the CMS
```

Comment:
Is the dash an English thing? I'm not familiar with it

Reply:
Yeah, the hyphen shows that the two words are related; so the synonyms are based in YML, not based AND YML. It's hard to explain with this particular example 😅